src/modules/workspace/create.ts

```diff
@@ -16,31 +25,87 @@ const warmUpRouteMap = async (workspace: string) => {
   } catch (err) {} // eslint-disable-line no-empty
```

Comment:
It's ok to no handle this error ?

Reply:
I think in this case it is - that wasn't changed from the original code. This code sends a request to the builder-hub on the recently changed workspace so Apps will warm up the route mapping (Apps has a route map for all workspaces - this is usually cached)

Reply:
The user doesn't have to wait for this request to wait, so I think it's ok to not handle this error